src/component/McardContainer/mcardcontainer.component.jsx

```diff
@@ -1,12 +1,31 @@
 import './mcardcontainer.css'
-import moviePosters from '../../data/movie_posters'
 import Mcard from '../Mcard/mcard.component'
 import { Fragment } from 'react'
+import {useState} from "react";
+import {useEffect} from "react";
 
 const McardContainer = ({pageEventHandler,mIdSetter}) => {
+  const [moviesData,setMoviesData ] = useState([]);
+
+  useEffect (()=>{
+
+    const fetchData = async ()=> {
+      try {
+        const moviesResponse = await fetch(
+          `https://rancid-tomatillos-api-cc6f59111a05.herokuapp.com/api/v1/movies`
+        )
+        const movies = await moviesResponse.json()
+        setMoviesData([...moviesData, ...movies])
+      } catch (err) {
+        console.log(err, "err")
+      }
+    }
+    fetchData()
+  },[])
```

Comment:
Really nice job of utilizing the useEffect in this code. Thank you for taking on the challenge and doing await with me